<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Stefan BÃ¶hmann</author>
 <class>PreferencesGeneralConfigPage</class>
 <widget class="QWidget" name="PreferencesGeneralConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Behavior on Application Startup</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>When Knipptasch starts:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="startupCombo"/>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="splashScreen">
        <property name="text">
         <string>Show splash screen on startup</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="defaultAccountFileLabel">
        <property name="text">
         <string>My default account file:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="defaultAccountFileRequesterLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="defaultAccountFileRequesterButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Tabbed Browsing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="hideTabBarIfEmpty">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Hide the tab bar when only one tab is open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showCloseButtonOnTabs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Show close button on each tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showCornerCloseButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Show corner close button</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="middleClickCloseTab">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Middle-click on a tab to close it</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="movableTabs">
        <property name="text">
         <string>Movable tabs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
